packages/components/avatar/src/avatar-group.tsx

```diff
@@ -0,0 +1,63 @@
+import {
+  Children,
+  cloneElement,
+  DetailedReactHTMLElement,
+  ReactNode,
+} from 'react'
+import { avatar } from '@mochi-ui/theme'
+import Avatar, { AvatarProps } from './avatar'
+
+const {
+  avatarGroupWrapperCva,
+  avatarGroupCva,
+  avatarGroupItemClsx,
+  avatarGroupItemAvatarClsx,
+} = avatar
+
+export type AvatarGroupProps = {
+  children: ReactNode
+  size?: 'xs' | 'sm' | 'base' | 'lg'
```

Comment:
`base` should be changed to `md`. And make sure `md` is the default size.

Reply:
I know this is from the `avatar` component. If you don't mind, please open a PR to fix the avatar.